### DISCUSSION TITLE
2023年终总结 - 你好2024！ | 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#29] [General]
# 2023年终总结 - 你好2024！ | 倾旋的博客

2023年转眼过去，好像2022年-2023年很少写一写总结了，心血来潮给博客装修了一下，为了迎接2024年的到来，认真写一篇年度总结，算是给这一年一个交代，我会介绍一下这两年以来自己做的一些事情和成长。

https://payloads.online/archivers/2023/12/29/8b9c967a-49ef-4f1a-8ea5-ccd19b9a708e

**Comment by yangzhao917** [comment]
11

**Comment by Nshimmer** [comment]
华晨宇《美好的事可不可以发生在我身上》，大佬必须有品！！！！！！！！

**Reply by Rvn0xsy** [reply]
华晨宇的没听过，我听的是原唱，感觉还挺好听的 :)

**Reply by Nshimmer** [reply]
大佬也可以听听华晨宇版本，非常nb

**Comment by TsVan1** [comment]
膜拜大佬

**Comment by JiangnanPsalter** [comment]
大佬，新年快乐

**Comment by Rvn0xsy** [comment]
新年快乐！

JiangnanPsalter ***@***.***> 于2024年2月20日周二 18:37写道：

**Comment by Bryan-Project** [comment]
想和大佬学习安全技术！能否方便加个vx呀～

**Comment by Rvn0xsy** [comment]
你的邮箱发我

Bryan-Project ***@***.***> 于2024年5月14日周二 16:55写道：

**Reply by Bryan-Project** [reply]
bryan09aws@outlook.com
感谢~

**Reply by LonFe627513141** [reply]
我也想加入你们，但我不善言辞

**Comment by yuyuana** [comment]
谢谢您，让我在一个普通的工作日下午感受到了人生的意义

**Reply by Rvn0xsy** [reply]
不客气，烂漫又璀璨的人生，你也值得 :)
